CMakeLists.txt

```diff
@@ -20,6 +20,11 @@ if(NOT CMAKE_BUILD_TYPE)
         )
 endif()
 
+if(WIN32)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_WIN32_WINNT=0x0601")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_WIN32_WINNT=0x0601")
+endif()
```

Comment:
Can you add a comment to the CMake file explaining why the flag is set?

Reply:
Comment added.